docs/index.jsx

```diff
@@ -14,20 +14,21 @@ import '../node_modules/@schibstedspain/sui-multimedia/dist/_sui-multimedia.scss
 const images = [{
   src: 'https://scontent-mad1-1.cdninstagram.com/t51.2885-15/e15/11199623_633712610062793_1285693904_n.jpg',
   type: 'image',
+  alt: 'Test',
   link: 'https://www.instagram.com/p/TNUdPKpMgM/?taken-by=davecarter'
 }];
 
 ReactDom.render(
     <SuiCard
-      primary={[
+      primary={
```

Comment:
As commented before, we'd need to put a wrapper in order to accept more than one component for each prop (primary & secondary).